### PR TITLE
Add skiff.com

### PR DESCRIPTION
**Pull request by nishils**
Skiff is a new privacy-oriented collaboration suite that offers private email and cloud storage services.

- [x] I have ran the SVG through SVGOMG to compress it down as much as possible.
- [x] Global rank is 68,312 (https://www.similarweb.com/website/skiff.com/#overview)
- [x] All links are in HTTPS.

...and y'all are awesome for maintaining this site!

### CODE REVIEW

File: img/s/skiff.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
... 244.969-157.879 261.271-20.164 2.575-40.757 1.717-60.92-2.145 13.728 2.145 27.457 2.574 40.756 1.716Zm60.492-.858c44.189-11.583 84.087-38.182 112.402-73.791 60.921-74.649 63.066-186.622 3.862-263.416-1.716-2.575-5.149-6.436-7.294-9.01-2.145-2.574-5.148-6.435-7.722-8.58-43.76-50.195-123.986-56.201-172.465-8.152 47.192-51.91 130.85-46.334 176.755 4.291 2.145 2.145 5.577 6.006 7.722 8.58 2.146 2.574 5.578 6.435 7.294 9.009 102.106 132.567 22.738 326.483-143.292 348.363-23.167 3.003-46.334 2.145-69.072-2.574 30.889 5.148 62.208 3.432 91.81-4.72Zm-81.514 10.726c16.732 2.574 33.893 3.861 51.054 3.003 144.149-6.435 249.259-138.144 223.089-280.149-8.581-46.333-31.319-89.664-64.353-123.128-2.574-2.574-5.148-5.148-8.151-7.293-52.769-45.905-135.141-44.189-184.049 6.864 48.05-54.485 133.425-57.488 187.481-11.154 3.003 2.145 6.006 5.148 8.58 7.722 23.596 23.167 42.044 51.482 54.485 81.943.859 2.145 2.575 6.864 3.433 9.009 1.287 3.432 3.003 9.009 3.861 12.441.858 3.004 1.716 6.436 2.574 9.439l1.287 6.435c7.293 33.893 7.293 69.501-.429 103.393-27.457 125.703-153.159 208.074-278.862 181.475Z"/></g><defs><clipPath id="a"><path fill="#fff" d="M.695.489h512.14v514.82H.695z"/></clipPath></defs></svg>
```
Comment (hkamran80) [inline img/s/skiff.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 513 516"><path d="M332.8 27.4c6 0 12 .4 18.4.9l4.7.4 4.7.9 9 1.7c2.6.4 5.2 1.3 7.8 2.1a94 94 0 0 1 14.1 5.2c2.6 1.2 6 2.5 8.6 3.8A259.1 259.1 0 0 0 67.6 85c-2.1 2.1-6.8 8.1-9 10.3a234.1 234.1 0 0 0-22.3 32.6l-6 10.7c-1.3 3-4.7 9.5-6 12L20 160.8l-3.9 10.8-4.2 12.8a163 163 0 0 0-3.5 13.3c-.4 1.7-2.1 9.5-2.6 11.2L4.1 220a242 242 0 0 0-2.5 61c.4 4.2 1.2 9.4 1.7 13.3l2.1 12.4 1.7 7.7.9 4.3c.8 3.4 2.1 8.6 3.4 12 .9 2.2 3 9.5 3.9 12 1.3 3 3 7.8 3.8 10.8 1.3 3 4.3 9.4 5.2 12.4l3.9 8.2.8 2 .4 1 .9.8 2.1 3.9c1.3 2.1 3 6 4.3 7.7L44 401l5.2 7.3c2.1 2.5 6 8.1 8.1 10.7 3.5 3.9 8.2 9.9 12 13.7l6.5 6.5 3.4 3L81 444a64 64 0 0 0 6 5.1c2.2 1.7 4.3 3.5 6 4.7.9.5 4.8 3.5 5.6 4 3 1.6 6.4 4.2 9.5 6 3 1.6 6.8 3.3 9.8 5a84 84 0 0 1 6 2.6c5.6 2.2 11.2 4.3 17.2 6 5.6 1.8 12 3 17.6 3.9 5.1.9 10.7 1.3 15.8 1.7h9.5c48-1.7 93-26.6 120.5-67.8.9-1.3 2.6-4.7 3.5-5.5 1.3-2.2 3-5.6 4.3-8.2l3-6c.8-2.6 2.5-6 3.4-8.2l1.7-4.2 1.3-4.3 1.3-4.3c2.1-7.3 3.4-14.6 4.3-22.3a146 146 0 0 0 1.3-19.3c-6 0-12.5-.9-18.5-1.8a147 147 0 0 1-73.8-31.3 152.5 152.5 0 0 1-54.9-148.4 153.3 153.3 0 0 1 151.5-124Zm-222.3 393c-3-2.2-6-5.2-8.6-7.7a244.7 244.7 0 0 1-54.4-82l-3.5-9c-.8-3.8-3-9-3.8-12.4l-2.6-9.5-1.3-6.4c-7.3-33.9-7.3-69.5.4-103.4C64.2 64.3 190-18 315.6 8.5A237.8 237.8 0 0 0 41.5 285.6a238 238 0 0 0 64.3 123.2 66 66 0 0 0 8.2 7.3c52.7 45.9 135.1 44.2 184-6.9-48 54.5-133.4 57.5-187.5 11.2Zm17.6-42.5c-2.1-2.1-5.6-6-7.7-8.6a99.2 99.2 0 0 1-7.3-9C11 227.3 90.4 33.8 256.4 11.5c22.7-2.5 46.3-1.7 69 2.6-30.4-4.7-61.3-3-90.9 5.2A214.7 214.7 0 0 0 122.1 93a214.5 214.5 0 0 0-4.3 263.5c1.7 2.5 5.2 6.4 7.3 9 2.2 2.5 5.2 6.4 7.7 8.6 43.8 50.1 124 56.2 172.5 8.1-47.6 52-130.9 46.3-177.2-4.3Zm47.2-15.9a90.4 90.4 0 0 1-9.9-8.1 193.3 193.3 0 0 1-48.9-74.7C73.6 162.2 150 33.8 274.5 18c20-2.6 40.7-1.7 60.8 2.1-13.3-1.7-27-2.6-40.3-1.7C144 27.8 60.3 196 145.3 321.3l3 4.3 3 3.8c3 3.9 6.4 8.2 9.4 11.6 36.5 43.3 102.1 55.8 148 20.2-37.3 32.6-94.4 30-133.4.8Zm26.6-40.7 3 3c2.6 2.1 5.6 4.7 8.2 6.4 1.2.9 4.2 2.6 5.1 3.5 2.6 1.2 4.7 3 7.3 4.2 1.3.5 4.3 2.6 5.6 3a103.6 103.6 0 0 0 77.6 4 99.9 99.9 0 0 1-29.6 7.2c-1.7 0-5.1.4-6.4.4h-6.4c-17.6-.8-34.8-6-50.2-15l-5.6-3.4-5.6-3.9c-2.1-1.3-4.3-3.4-6-4.7a174.2 174.2 0 0 1 73.8-298.2 171 171 0 0 1 69-2.5 177.2 177.2 0 0 0-100.3 18.4 172.3 172.3 0 0 0-39.5 277.6Z"/><path d="M200.2 184.9c-4.7-.9-9.9-1.3-15-1.8v.5a85 85 0 0 1 15 1.3c.4 0 .4 0 0 0Z"/><path d="M438.3 438.8c2.1-2.5 5.6-6 7.7-8.1 2.2-2.2 6.9-8.2 9-10.3a233.8 233.8 0 0 0 22.3-32.6c.9-1.7 5.2-9 6-10.7 1.3-3 4.8-9.5 6-12l4.3-10.4c.5-.8 3.5-9.4 3.9-10.7l4.3-12.9c1.3-4.2 2.6-9 3.4-13.3l2.6-11.1 1.7-11.2a264 264 0 0 0 1.7-60.9c-.4-4.3-1.3-9.4-1.7-13.3l-2.1-12.4-1.8-7.7-.8-4.3c-.9-3.5-2.2-8.6-3.4-12-.9-2.2-3-9.5-3.9-12-1.3-3-3-7.8-3.9-10.8-1.2-3-4.3-9.4-5.1-12.4l-3.9-8.2-.8-2.1-.5-.9-.4-.8-2.1-3.9c-1.3-2.1-3-6-4.3-7.7l-7.3-11.6-5.2-7.3c-2.1-2.6-6-8.1-8.1-10.7-3.4-3.9-8.2-9.9-12-13.8l-6.5-6.4-3.4-3-1.7-1.7a64 64 0 0 0-6-5.2l-6-4.7c-.9-.4-4.7-3.4-5.6-3.8-3-1.8-6.4-4.3-9.4-6-3-1.8-6.9-3.5-10-5.2a84 84 0 0 1-6-2.6 88.5 88.5 0 0 0-15-5.1c-2-.4-4.2-1.3-6.4-1.7-1.3-.5-5.1-1.3-6.4-1.3-2.1-1.3-4.7-1.7-6.4-2.2-5.2-.8-10.8-1.2-16-1.7h-9.3a150.3 150.3 0 0 0-120.2 67.8c-.8 1.3-2.5 4.7-3.4 5.6-1.3 2.1-3 5.6-4.3 8.1l-3 6c-.9 2.6-2.6 6-3.4 8.2l-1.7 4.3-1.3 4.3c-.5 1.3-.9 3-1.3 4.3-2.2 7.3-3.5 14.6-4.3 22.3-1.3 6.8-1.7 13.3-1.7 19.7 33.4 0 65.6 11.6 92.2 32.2a152.5 152.5 0 0 1 55 148.4 153.2 153.2 0 0 1-151.5 124.9c-6 0-12-.5-18.5-.9l-4.7-.4-4.7-.9-9-1.7c-2.6-.4-5.2-1.3-7.7-2.1A94 94 0 0 1 122 477c-2.6-1.3-6-2.6-8.6-3.8 0 0-.4-.5-.8-.5a259 259 0 0 0 325.6-33.9Zm-166.5 33a172.2 172.2 0 0 0 40.4-277.5l-3-3a91.5 91.5 0 0 0-8.2-6.4c-1.3-.9-4.3-2.6-5.1-3.5-2.6-1.3-4.8-3-7.3-4.3-1.3-.4-4.3-2.1-5.6-3a103.6 103.6 0 0 0-77.7-3.8A99.9 99.9 0 0 1 235 163c1.8 0 5.2-.5 6.5-.5h6.4c17.6.9 34.8 6 50.2 15 1.3 1 4.3 2.6 5.6 3.5l5.6 3.9c2.1 1.2 4.2 3.4 6 4.7a174.2 174.2 0 0 1-73.8 298.1 170.6 170.6 0 0 1-69.1 2.6c33.5 3.9 68.6-2.6 99.5-18.4Zm-52.7 25.4c150.6-9.5 234.2-177.6 149.3-303l-3-4.2-3-3.9a274 274 0 0 0-9.5-11.5c-36.4-43.4-102-55.8-148-20.2 37.3-32.6 94.4-30 133.4-.9a88 88 0 0 1 9.9 8.2 195.9 195.9 0 0 1-109 336 193.7 193.7 0 0 1-60.9-2.2c13.7 2.1 27.5 2.5 40.8 1.7Zm60.5-.9a215 215 0 0 0 116.3-337.2c-1.8-2.6-5.2-6.4-7.4-9-2.1-2.6-5.1-6.4-7.7-8.6-43.8-50.2-124-56.2-172.5-8.1 47.2-52 130.9-46.4 176.8 4.3 2.1 2.1 5.6 6 7.7 8.5 2.2 2.6 5.6 6.5 7.3 9 102.1 132.6 22.7 326.5-143.3 348.4a205 205 0 0 1-69-2.6 212 212 0 0 0 91.8-4.7ZM198 507c16.8 2.6 34 4 51.1 3a237.2 237.2 0 0 0 158.7-403.2 56.3 56.3 0 0 0-8.1-7.3c-52.8-46-135.2-44.2-184 6.8C263.6 52 349 49 403 95.2c3 2.1 6 5.1 8.6 7.7 23.6 23.2 42 51.5 54.5 82l3.4 9c1.3 3.4 3 9 3.9 12.4l2.5 9.4 1.3 6.5c7.3 33.9 7.3 69.5-.4 103.4C449.4 451.3 323.7 533.6 198 507Z"/></svg>
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/s/skiff.com.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "Skiff": {
+      "domain": "skiff.com",
+      "additional-domains": [
+        "bitkeep.app",
+        "ethereum.email",
+        "keplr.xyz",
+        "solana.email"
+      ],
```
Comment (hkamran80) [inline entries/s/skiff.com.json:9]: Why do you have these domains as additional domains?

Comment (nishils) [reply entries/s/skiff.com.json:9]: When a user with a crypto wallet links it to their skiff account, they gain access to a few additional domains that Skiff owns. An ethereum user gets access to ethereum.email for example.

Skiff owns these domains outright and can be confirmed via Whois records.

We can also remove if needed if this wasn't the intended use case of additional domains.

Comment (hkamran80) [reply entries/s/skiff.com.json:9]: Ah, gotcha. That's the intended use case of additional domains: to provide a list of alternate domains that are used by the service.

Comment (hkamran80) [reply entries/s/skiff.com.json:9]: Are you able to access those domains? I get 404s for all but Keplr, which redirects to the Keplr website.

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
Thank you! Your contribution will be visible in around 10 minutes, although it may take longer due to server and browser caches.

**Pull request merged**